pkg/cmd/secrets.go

```diff
@@ -109,6 +111,32 @@ $ doppler secrets download --format=env --no-file`,
 	Run:  downloadSecrets,
 }
 
+var secretsTemplateCmd = &cobra.Command{
+	Use:   "template <filepath>",
+	Short: "Inject secrets into a template file",
+	Long:  "Inject secrets into a template file. See https://golang.org/pkg/text/template/ for full syntax",
+	Example: `$ cat template.yaml
+{{- /* Full comment support */ -}}
+host: {{.API_HOST}}
+port: {{.API_PORT}}
+
+{{with .OPTIONAL_SECRET}}
```

Comment:
I wanted to call out this `with` syntax, which is awesome! It creates a new scope with the provided value so you can just refer to the secret as `.` inside the block.

Reply:
That's a good point. People who really want to power-use this feature can dig into the Go docs and learn about `with`.